src/ValidationsFactory.js

```diff
@@ -63,10 +63,9 @@ class FormNestedScreenValidations extends Validations {
       //the screen does not load
       return;
     }
-    for (const item of definition) {
-      if (item.items) {
-        await ValidationsFactory(item.items, { screen: this.screen }).addValidations(validations);
-      }
+
+    if (definition && definition[0] && definition[0].items) {
```

Comment:
At this point `definition` is not empty, because it was validated at line 62
```suggestion
    if (definition[0] && definition[0].items) {
```

Reply:
solved